test/test_group.py

```diff
@@ -248,6 +248,37 @@ def test_add_group_with_dot_in_pkgrel(db, client):
     set_and_assert_group_data(db, client, url_for('tracker.add_group'), affected='1.2-3.4')
 
 
```

Comment:
i would love to see a test as well that already has the lib32 variant and adds the non-lib32 one later. maybe that case should be detected and flashed as well :smiley_cat:

Reply:
Fixed, but the code is rather deeply nested, maybe some lambda/filter/map logic can be used to make it nicer.